### DISCUSSION TITLE
Build only specified routes

### DISCUSSION THREAD

**Discussion by EvanPartidas** [withastro/roadmap#506] [Proposal]
### Body

# Summary

I would like a way to update specified routes/files. This way, updating or adding a single markdown file does not cause the entire website to be rebuilt.

# Background & Motivation

Although SSR is available to generate more dynamic content, it can be overkill. In the case of a website with many routes and frequent changes to specific routes, such as a blog, there is no reason to generate each route dynamically, as the vast majority of blogposts will remain static. There is also no reason to rebuild the entire website when you only updated a single markdown file. 

# Goals

- Save computation power when using SSG with frequent, but small, modifications
- Create a way for updates to be made efficiently

**Comment by florian-lefebvre** [comment]
See https://github.com/withastro/roadmap/pull/763
